definitions/consented/json/CaseEventToFields/CaseEventToFields-respSol-nonprod.json

```diff
@@ -31,7 +31,7 @@
     "CaseEventID": "FR_updateContactDetails",
     "CaseFieldID": "RespondentOrganisationPolicy",
     "PageFieldDisplayOrder": 27,
-    "DisplayContext": "MANDATORY",
+    "DisplayContext": "COMPLEX",
```

Comment:
needs to add this event FR_updateContactDetails to CaseEventToComplexType, alike FR_solicitorCreate